launcher/main/main.c

```diff
@@ -166,7 +166,7 @@ static void retro_loop(void)
 
```

Comment:
This whole file should also be undone, the reason for these #ifdefs are backwards compatibility, they aren't needed on new devices